packages/atomic/src/components/facets-v1/facet-search/facet-search-matches.tsx

```diff
@@ -0,0 +1,35 @@
+import {FunctionalComponent, h} from '@stencil/core';
+import {i18n} from 'i18next';
+
+interface FacetSearchMatchesProps {
+  i18n: i18n;
+  query: string;
+  numberOfMatches: number;
+  hasMoreMatches: boolean;
+}
+
+export const FacetSearchMatches: FunctionalComponent<FacetSearchMatchesProps> = (
+  props
+) => {
+  if (!props.numberOfMatches) {
+    return (
+      <div class="ellipsed p-3 bg-neutral-light text-neutral-dark text-sm">
+        {props.i18n.t('noMatchesFoundFor', {
+          query: props.query,
+          interpolation: {escapeValue: false},
```

Comment:
Since we just put what the user just wrote into the input, it looks weird to sanitize the display when there is a use of special characters

Reply:
But if you write `<script>alert('foo')</script>` in the facet search box and there's no result, what happens ?

I'm guessing it's correctly escaped with JSX ?

Reply:
Yes exactly